holmes-py/specs/gdc_data_portal_v2/project_summary/validate_tables.spec

```diff
@@ -0,0 +1,263 @@
+    |f1a3cd15-5978-4ae8-9c30-8d5ebc4c000b   |
+* Verify that "JSON from Project Summary Annotations" does not contain specified information
+    |required_info                          |
+    |---------------------------------------|
+    |007eafca-85e4-4924-ba08-543a12167217   |
+    |727aded8-f047-485f-b9cc-edd54aa940af   |
+    |e00a1454-0687-48a1-a6c9-caeef0bc4221   |
+    |5a88652c-cc2f-44c8-9e61-e89d46222743   |
+    |d91781d2-4493-4321-ab88-4ec4ac6a9afb   |
+    |e6ac9549-7257-4733-ad48-048e991f8bcb   |
+
+## Annotations Table - Validate JSON File Fields
+  |field_name                               |
+  |-----------------------------------------|
+  |annotation_id	                        |
+  |entity_submitter_id	                    |
+  |notes	                                |
+  |entity_type	                            |
+  |case_id	                                |
+  |project.project_id                       |
+  |project.program.name                     |
+  |classification	                        |
+  |entity_id	                            |
+  |category	                                |
+  |created_datetime                         |
+  |status                                   |
+  |case_submitter_id	                    |
+* Verify that the "JSON from Project Summary Annotations" has <field_name> for each object
+
+## Validate Created Cohorts
+* Navigate to "Downloads" from "Header" "section"
+* Switch cohort to "CPTAC-3 Primary Site Pancreas" from the Cohort Bar dropdown list
+* "CPTAC-3 Primary Site Pancreas" should be the active cohort
+* Collect Cohort Bar Case Count for comparison
+* Collect "Files" Count on the Repository page
+* Verify "CPTAC-3 Primary Site Pancreas Case Count" and "Cohort Bar Case Count" are "Equal"
+* Verify "CPTAC-3 Primary Site Pancreas File Count" and "Files Count Repository Page" are "Equal"
+
+## Validate Combinations - Number of Diseases, Primary Sites, Annotations
+* Quick search for "TCGA-LGG" and go to its page
+* Is text "Cases and File Counts by Data Category" present on the page
+* Verify the table "Summary Project Summary" is displaying this information
+    |text_to_validate                       |
+    |---------------------------------------|
+    |Disease Type                           |
+    |Primary Site                           |
+    |Gliomas                                |
+    |Brain                                  |
+* Verify these items are not on the page
+    |item_to_validate                               |
+    |-----------------------------------------------|
+    |Table Primary Sites Project Summary            |
+* Quick search for "MATCH-C1" and go to its page
+* Is text "Cases and File Counts by Experimental Strategy" present on the page
+* Verify these items are not on the page
+    |item_to_validate                               |
+    |-----------------------------------------------|
+    |Table Annotations Project Summary              |
```

Comment:
Have a question on this line. You say that Annotations and Project Summary should be on this page for this project. I understand that Annotations is not on this projects page, But the "Project Summary" exists right?

Reply:
Yeah this means the annotations table should not be present on that particular project summary page